### PR TITLE
feat(roll): roll Playwright to 1.13.0-next-1625813701000

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

**Review by yury-s — APPROVED** [review]

**Comment by yury-s** [comment]
The bots are unhappy

File: tests/async/test_download.py [hunk 1]
```diff
@@ -349,3 +349,20 @@ async def test_should_delete_downloads_on_browser_gone(browser_factory, server):
     assert os.path.exists(path1) is False
     assert os.path.exists(path2) is False
     assert os.path.exists(os.path.join(path1, "..")) is False
+
+
+async def test_download_cancel_should_work(browser, server):
+    def handle_download(request):
```
Comment (kumaraditya303) [inline tests/async/test_download.py:355]: Consider using context managers in new test code

File: tests/async/test_page_base_url.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+# limitations under the License.
+
+from playwright.async_api import Browser, BrowserType
+from tests.server import Server
+
+
+async def test_should_construct_a_new_url_when_a_base_url_in_browser_new_context_is_passed(
+    browser: Browser, server: Server
+):
+    context = await browser.new_context(base_url=server.PREFIX)
```
Comment (kumaraditya303) [inline tests/async/test_page_base_url.py:22]: ditto

File: tests/async/test_download.py (hunk 1 above)
Comment (mxschmitt) [reply tests/async/test_download.py:355]: I want to keep the tests similar to upstream tests, so its easier to work with them in the future.

**Pull request merged**